### DISCUSSION TITLE
如何安装密钥

### DISCUSSION THREAD

**Discussion by ChudieGu** [MuiseDestiny/zotero-gpt#121] [Q&A] [answered]
小白发问：点击askPDF之后显示需要secretKEY，但是不知道是要贴在哪里呀？以及secretKEY就是chatgpt账号里的API吗
<img width="347" alt="微信截图_20230425161353" src="https://user-images.githubusercontent.com/131753758/234216111-ceeb21c8-db60-4c65-9f72-4bdcb99b91bb.png">

**Comment by MuiseDestiny** [comment, marked as answer]
readme和置顶issue可以解决你的问题。如果还有疑问提issue
